core/core-load-paths.el

```diff
@@ -18,7 +18,7 @@
 ;; paths
 (defvar spacemacs-start-directory
   user-emacs-directory
-  "Spacemacs start directory.")
+  "Spacemacs start directory.  This is set within `init.el'.")
```

Comment:
I'm a bit lost here.

`spacemacs-start-directory` is set in dotspacemacs. But in `init` file